ant-node-manager/src/cmd/node.rs

```diff
@@ -128,7 +127,7 @@ pub async fn add(
         enable_metrics_server,
         evm_network: evm_network.unwrap_or(EvmNetwork::ArbitrumOne),
         env_variables,
-        relay,
+        relay: false,
```

Comment:
shall the struct of `AddNodeServiceOptions` also got the `relay` field removed ?